format/format.go

```diff
@@ -278,16 +282,25 @@ func (rf RestoreFlags) HasSpacesAroundBinaryOperationFlag() bool {
 	return rf.has(RestoreSpacesAroundBinaryOperation)
 }
 
+func (rf RestoreFlags) HasWithDefaultDB() bool {
+	return rf.has(RestoreWithDefaultDB)
```

Comment:
IMO returning `rf.DefaultDB != ""` is enough, no need to introduce a flag

Reply:
Done.